composer.json

```diff
@@ -1,5 +1,8 @@
 {
 	"name": "dg/adminer-custom",
 	"description": "Customization for Adminer, the best database management tool written in PHP.",
-	"keywords": ["database", "mysql", "postgresql", "sqlite", "mssql", "oracle", "mongodb", "elasticsearch"]
+	"keywords": ["database", "mysql", "postgresql", "sqlite", "mssql", "oracle", "mongodb", "elasticsearch"],
+	"require": {
+		"vlucas/phpdotenv": "^2.4"
```

Comment:
I think it would be better to use [symfony/dotenv](https://symfony.com/doc/current/components/dotenv.html).

Reply:
Fixed